### ISSUE TITLE
Improve granularity of Heel issues

### ISSUE THREAD

**Issue by ericaVoss** [OHDSI/Achilles#175]
I recently got these two errors from ACHILLES:

`ERROR: 706 - Distribution of age by drug_concept_id (count = 2); min value should not be negative`
`ERROR: 906 - Distribution of age by drug_concept_id (count = 2); min value should not be negative`

Looking at the SQL it is easy to tell one is for DRUG_EXPOSURE ([706](https://github.com/OHDSI/Achilles/blob/d0de5c2882e4ebac9fb384bd1f590b24cd3b848b/inst/sql/sql_server/Achilles_v5.sql#L3680)) and the other for DRUG_ERA ([906](https://github.com/OHDSI/Achilles/blob/d0de5c2882e4ebac9fb384bd1f590b24cd3b848b/inst/sql/sql_server/Achilles_v5.sql#L4593)).  

It will be helpful if the descriptions called this out so you don't have to go to the SQL to figure it out.  Thank you!

**Comment by vojtechhuser** [comment]
can you please include the rule_id for those errors. 

I proposed breaking some 'umbrella rules' into more granular (managable ones) (and making the error message more specific to the issue). But there was no agreement on doing this and I don't want to surprise the users or the team too much.

http://forums.ohdsi.org/t/heel-number-of-death-records-outside-valid-observation-period-and-umbrella-data-quality-rules/1859

@ericaVoss , could you please contribute your thoughts to the forum thread above.

**Comment by vojtechhuser** [comment]
I will rename the issue to also include the larger problem of umbrella rules

**Comment by vojtechhuser** [comment]
forum replies indicate that some umbrella rules produce output that is hard to follow.

**Comment by vojtechhuser** [comment]
This is because there are umbrella rules that check for a lot of things in one query. The only approach for more specific messages is to break them into more rules.

**Comment by vojtechhuser** [comment]
In order to turn on of off output in Fetch Heel, umbrella rules are too broad. (discussed with David B.)

**Comment by fdefalco** [comment]
Heel has been deprecated in favor of the DataQualityDashboard.
